internal/api/v1/handlers/job.go

```diff
@@ -82,10 +82,11 @@ func (h *JobHandler) ListJobs(c *fiber.Ctx) error {
 // CreateJob handles the request to create a new job
 func (h *JobHandler) CreateJob(c *fiber.Ctx) error {
 	var req struct {
-		Name        string                           `json:"name"`
-		ProjectName string                           `json:"project_name"`
-		WebhookURL  string                           `json:"webhook_url"`
-		Instances   []infrastructure.InstanceRequest `json:"instances"`
+		JobName      string                           `json:"job_name"`
+		InstanceName string                           `json:"instance_name"`
+		ProjectName  string                           `json:"project_name"`
+		WebhookURL   string                           `json:"webhook_url"`
+		Instances    []infrastructure.InstanceRequest `json:"instances"`
```

Comment:
This one also could be extracted

Reply:
same, I definitely did this somewhere. I'll find it.